Source/AwsCommonRuntimeKit/mqtt/Mqtt5Packets.swift

```diff
@@ -287,11 +339,67 @@ public class Subscription {
         self.noLocal = noLocal
         self.retainAsPublished = retainAsPublished
         self.retainHandlingType = retainHandlingType
+
+        aws_byte_buf_clean_up(&topicFilterBuffer)
+        _ = self.topicFilter.withByteCursor { topicFilterCursor in
+            aws_byte_buf_init_copy_from_cursor(&topicFilterBuffer, allocator, topicFilterCursor)
+        }
+    }
+
+    private var topicFilterBuffer = aws_byte_buf()
+
+    typealias RawType = aws_mqtt5_subscription_view
+    func withCStruct<Result>(_ body: (RawType) -> Result) -> Result {
+        var view = aws_mqtt5_subscription_view()
+        view.qos = self.qos.nativeValue
+        view.no_local = self.noLocal ?? false
+        view.retain_as_published = self.retainAsPublished ?? false
+        if let _retainType = self.retainHandlingType {
```

Comment:
What does _variableName naming scheme represent? Maybe just get rid of _?
```suggestion
        if let retainType = self.retainHandlingType {
```